code/game/objects/items/stacks/barbed_wire.dm

```diff
@@ -30,19 +30,6 @@
 /obj/item/stack/barbed_wire/attackby(obj/item/I, mob/user, params)
 	. = ..()
 
-	if(!istype(I, /obj/item/stack/rods))
```

Comment:
You can just cut the rest of this proc since razorwire making was the only thing it did.